packages/sv/lib/core.ts

```diff
@@ -0,0 +1,75 @@
+// from externals
+export { log } from '@clack/prompts';
+export { default as dedent } from 'dedent';
+export * as Walker from 'zimmerframe';
+
+// from internals
+export { defineAddon, defineAddonOptions } from './core/addon/config.ts';
+export { color } from './cli/add/utils.ts';
+// TODO JYC: move to utils all these bellow?
+export { isVersionUnsupportedBelow } from './core/common.ts';
+export { fileExists } from './cli/add/utils.ts';
+export { resolveCommand } from 'package-manager-detector/commands';
+export { getNodeTypesVersion, addToDemoPage } from './addons/common.ts';
+// from internals, in utils
+import { createPrinter } from './core/utils.ts';
+export const utils = {
+	createPrinter: createPrinter as typeof createPrinter
+};
+
+// parsing & languages
+export * as css from './core/tooling/css/index.ts';
+export * as js from './core/tooling/js/index.ts';
+export * as html from './core/tooling/html/index.ts';
+import { ensureScript, addSlot, addFragment } from './core/tooling/svelte/index.ts';
+/**
+ * Helper functions to manipulate Svelte code.
+ */
+export const svelte = {
+	ensureScript: ensureScript as typeof ensureScript,
+	addSlot: addSlot as typeof addSlot,
+	addFragment: addFragment as typeof addFragment
+};
```

Comment:
Why is this not the same as for the other langs? I thought they were setup identically

Reply:
It was a test, yes!
Doing this enable to have `JSDoc` comments that can be used in doc! and in code when user will import `svelte` 👍

So in `core.ts` we should never export `*`. I did only a few JSDoc test and wanted to speak with you do to more ^^